Sources/ArcGISToolkit/Common/AttachmentPreview.swift

```diff
@@ -204,7 +204,7 @@ struct AttachmentPreview: View {
                             .padding(8)
                     }
                 }
-                if attachmentModel.attachment.loadStatus != .loaded {
+                if attachmentModel.loadStatus != .loaded {
```

Comment:
Same for the changes in this file: not directly related to the bug; should've been accessing the model's property.